pom.xml

```diff
@@ -177,7 +177,6 @@
                 <groupId>jakarta.validation</groupId>
                 <artifactId>jakarta.validation-api</artifactId>
                 <version>${version.jakarta-validation}</version>
-                <scope>provided</scope>
```

Comment:
Sorry, I should have asked this in the prev review, why are we removing provided ?

Reply:
sorry, that was an accidental change